packages/editor-ui/src/components/ParameterInput.vue

```diff
@@ -709,7 +721,9 @@ export default mixins(
 			getPlaceholder(): string {
 				return this.isForCredential
 					? this.$locale.credText().placeholder(this.parameter)
-					: this.$locale.nodeText().placeholder(this.parameter, this.path);
+					: this.inputPlaceholder
+						? this.inputPlaceholder
+						: this.$locale.nodeText().placeholder(this.parameter, this.path);
```

Comment:
maybe wrap this in parenthesis for clarity
```suggestion
					: (this.inputPlaceholder
						? this.inputPlaceholder
						: this.$locale.nodeText().placeholder(this.parameter, this.path));
```

Reply:
Agree.